src/praisonai/praisonai/cli/commands/managed.py

```
@@ -532,6 +532,97 @@ def ids_restore(
+    if not force:
+        confirm = typer.confirm(f"Delete session {session_id}?")
+        if not confirm:
+            typer.echo("Cancelled.")
+            return
+
+    client = _get_client()
+    try:
+        # For Anthropic managed agents, delete via API
+        response = client.delete(f"/v1/agent_sessions/{session_id}")
+        typer.secho(f"Session {session_id} deleted", fg="green")
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)